src/freenas/usr/local/lib/middlewared_truenas/alert/source/failover_sync.py

```diff
@@ -12,7 +12,7 @@ class FailoverSyncFailedAlertClass(AlertClass, SimpleOneShotAlertClass):
     level = AlertLevel.CRITICAL
     title = "Automatic Sync to Peer Failed"
     text = (
-        "Failed to sync configuration information to standby storage "
+        "Failed to sync configuration information to passive storage "
```

Comment:
PM had previously requested standardizing the controller terminology from active/passive to active/standby: https://jira.ixsystems.com/browse/NAS-102356?jql=issuetype%20%3D%20Epic%20AND%20text%20~%20%22standby%22 . Things might have changed since then, but pointing it out in case it got fixed the wrong direction.

Reply:
Well that's frustrating....but clearly it needs to go the other way. @sonicaj sorry, you'll have to change it back to `standby` 😛

Reply:
It was a quick fix, dropping that commit did the job ; )